src/v2/guide/conditional.md

```diff
@@ -196,4 +196,6 @@ Generally speaking, `v-if` has higher toggle costs while `v-show` has higher ini
 
 ## `v-if` with `v-for`
 
+> Use v-if and v-for together is **not recommended**. See the style guide for futher infomation [Avoid v-if with v-for](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential)
```

Comment:
I'd suggest something like this instead:

```md
> Use v-if and v-for together is **not recommended**. See the [style guide](/v2/style-guide/#Avoid-v-if-with-v-for-essential) for further information.
```

Notice the wording changes and the two spelling mistake corrections.

Reply:
thanks for your change and correcting, your version is much better.